### PR TITLE
Docs: Split SSO docs into HowTo/Explanation

### PR DESCRIPTION
**Pull request by ericholscher**
This is an initial attempt.
I ended up making a couple decisions:

* Breaking each SSO method out into its own page, to make it clearer and rank better on Google
* Pulling the content around generic team membership out into its own page, since it wasn't really Google related

There's probably some light finishing touches to clean things up, 
but wanted to get this up today.

### CODE REVIEW

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your Organization's settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`Username dropdown` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:21]:
```suggestion
* Navigate to :guilabel:`<Username dropdown> 🔽️` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
```

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
+
+User setup
+~~~~~~~~~~
+
+Using this setup,
+all users who have access to the domain you configure will be granted a subset of permissions on your organization automatically on account creation.
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:31]:
```suggestion
all users who have access to the configured Google Workspace will be granted a subset of permissions on your organization automatically on account creation.
```

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Other users can also be added to your organization and given access in an ad hoc manner.
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:32]:
```suggestion
.. tip::

   You can still add outside collaborators and manage their access. Organizations may for instance wish to give read-access to onboarding documents or grant temporary write access to external consultants.
   
.. seealso::

   :doc:`/commercial/sharing`
     Read more about methods for sharing **read** access to private sets of documentation without requiring a login for Read the Docs or your SSO provider.
```

This is a really good tip. This is like the real advantage of using SSO only for authentication, since it gives the flexibility of adding outside guest contributors and readers very easily.

Perhaps worth also mentioning secret share links?

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Revoke user's access to all the projects
+----------------------------------------
+
+By disabling the Google Workspace account with email ``employee@company.com``,
+you revoke access to all the projects that user had access and disable login on Read the Docs completely for that user.
+
+.. seealso::
+    * Additional user management options are available at :doc:`/guides/manage-read-the-docs-teams`.
+    * To learn more about choosing a Single Sign-on approach,
+    please read :doc:`/commercial/single-sign-on`.
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:70]:
```suggestion
.. seealso::

   :doc:`/guides/manage-read-the-docs-teams`
     Additional user management options
   :doc:`/commercial/single-sign-on`
     Information about choosing a Single Sign-on approach
```

File: docs/user/commercial/single-sign-on.rst [hunk 5]
```diff
@@ -1,5 +1,5 @@
-Single Sign-On
-==============
+Choosing a Single Sign-On approach
+==================================
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/commercial/single-sign-on.rst:2]: Seems like it's back to a "reference" type title, instead of explanation?

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+To change your Organization's settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`Username dropdown` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:25]: Heh, mostly just to give those people an obvious next step.

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+User setup
+~~~~~~~~~~
+
+Users in your organization must have their GitHub, Bitbucket, or GitLab
+:doc:`account connected </connected-accounts>`,
+otherwise they will lose access to all repositories.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:32]: What part don't you understand? If they don't have their account connected, they won't have access to anything on RTD, because that's how auth is handled.

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+------------
+
+You can enable this feature in your organization by:
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Username name>` > :guilabel:`Settings` > :guilabel:`Authorization`
+* Select :guilabel:`GitHub, GitLab or Bitbucket` on the *Provider* dropdown.
+* Select :guilabel:`Save`
+
+The users in your organization **must connect**  their GitHub, Bitbucket or GitLab
+:doc:`account connected </connected-accounts>`,
+otherwise they will lose access to all repositories.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:44]: Yes, but it's important.

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+You can read more about `granting permissions on GitHub`_ in their documentation.
+
+.. warning:: Once you enable this option, your existing Read the Docs teams will not be used.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:47]: Yea. I don't see the change in your suggestion?

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+you are giving the user access to read the documentation *and* to be an administrator of the associated project on Read the Docs.
+
+Grant access to import a project
+--------------------------------
+
+When SSO with a VCS provider is enabled, only owners of the Read the Docs organization can import projects.
+Adding users as owners of your organization will give them permissions to import projects.
+
+To be able to import a project,
+that user must have **admin** permissions in the VCS repository associated.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst]: > Side note: Isn't "Git" supposed to be a proper noun?

Yea, I think so.. We should have this in our style guide, I don't think we're doing it consistently.

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+Revoke access to a project
+--------------------------
+
+If a user should not have access to a project,
+you can revoke access to the git repository,
+and this will be automatically reflected in Read the Docs.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:81]: Don't think your change is better here. It's less explicit, and the first part isn't a complete thought.

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 1 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst:21]: Hrm, we're using this in a bunch of places, but I do think it's a good addition.

File: docs/user/commercial/single-sign-on.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/commercial/single-sign-on.rst:2]:
```suggestion
Choosing a Single Sign-On approach for your organization
=========================================================
```

[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/commercial/single-sign-on.rst:2]: I think that "Choosing a Single Sign-On approach for organizations" is too long but it's very important to capture that it's _for organizations_? There is already SSO for anyone who has a fundamental understanding of RTD, so the point of the explanation is easily missed if we don't include this.

All Explanation articles are already nested in Explanation which gives the titles that context.

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 3 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst]: Yea, I thought about sharing here, so definitely a good addition. `seealso` might be too heavy inline, but we'll see how it renders.

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 1 above)
Comment (benjaoming) [reply docs/user/guides/setup-single-sign-on-google-email.rst:21]: I thought of the emoji as an experiment, I haven't seen how it looks. So in case you apply it, we can decide what to do after?

File: docs/user/commercial/single-sign-on.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/commercial/single-sign-on.rst:2]: The long suggestion can work as well :+1: I think it's great to start with a long title that hits the nail and then review suggestions of shortening it after :+1:

File: docs/user/commercial/single-sign-on.rst [hunk 12]
```diff
@@ -18,123 +18,26 @@ Users can log out by using the :ref:`Log Out <versions:Logging out>` link in the
...
-Instead of revoking access completely, just need lower down permissions to **read** only.
+that user just needs to be granted permissions in the git repository associated with the project.
 
+Once you enable this option,
+your existing Read the Docs teams will not be used.
+All authentication will be done using your git provider,
+including any two-factor authentication and additional Single Sign-on that they support.
 
 SSO with Google Workspace
 -------------------------
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/commercial/single-sign-on.rst:36]: > Should we also mention the "email domain" method? I.e. that someone with the correct email domain can sign up and automatically enroll in the matching org?

It seems that is what we're discussing? 

> Is "no SSO" also supported for organizations? I.e. that if someone needs the simplest and fastest method, then they don't need to configure anything.

Updated to clarify that in the intro.

> Final and last idea... "Don't see your organization's Single Sign-On method?" with the intention that a corporate user may contact us and to make it evident that SSO support is extended through demand (or active contribution!)?

Added 👍

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 1 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst:21]: I'll leave it off for now, but it looks reasonable:

Screenshot 2022-12-13 at 2 03 47 PM

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (humitos) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:32]:
```suggestion
otherwise they won't have access to any project on Read the Docs.
```

I think that sounds less scary 😄

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (humitos) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst]: We've talked about this already, and I think these things are good to be automatically validated with vale or similar. 

* https://github.com/readthedocs/readthedocs.org/issues/5876
* https://github.com/readthedocs/readthedocs.org/issues/8108

File: docs/user/guides/manage-read-the-docs-teams.rst [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+* Navigating to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > :guilabel:`Teams` > :guilabel:`<Team name>`
+* Clicking :guilabel:`Remove` next to the user.
+
+
+Grant access to users to import a project
+-----------------------------------------
+
+Make the user a member of any team with *admin* permissions,
+they will be granted access to import a project.
```
Comment (humitos) [inline docs/user/guides/manage-read-the-docs-teams.rst:34]:
```suggestion
they will be granted access to import a project on that team.
```

Maybe?

File: docs/user/guides/manage-read-the-docs-teams.rst [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+* Navigating to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > :guilabel:`Teams` > :guilabel:`<Team name>`
+* Clicking :guilabel:`Invite Member`.
+* Input the user's Read the Docs username or email address.
+* Clicking :guilabel:`Add member`.
+
+Removing a user from a team
+----------------------------
+
+Removing a user to a team removes all permissions that team gave them
```
Comment (humitos) [inline docs/user/guides/manage-read-the-docs-teams.rst:24]:
```suggestion
Removing a user from a team removes all permissions that team gave them
```

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your :ref:`Organization <commercial/organization:Organizations>`'s settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
```
Comment (humitos) [inline docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:22]:
```suggestion
* Look at the :guilabel:`Owners` section on the right menu.
```

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+
+.. _granting permissions on GitHub: https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization
+
+Enabling SSO
+------------
+
+You can enable this feature in your organization by:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > :guilabel:`Settings` > :guilabel:`Authorization`
+* Select :guilabel:`GitHub, GitLab or Bitbucket` on the *Provider* dropdown.
```
Comment (humitos) [inline docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:43]:
```suggestion
* Select :guilabel:`GitHub, GitLab or Bitbucket` on the :guilabel:`Provider` dropdown.
```

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+To change your :ref:`Organization <commercial/organization:Organizations>`'s settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
```
Comment (humitos) [inline docs/user/guides/setup-single-sign-on-google-email.rst:25]: It seems we are repeating this text in multiple files. It may makes sense to convert it into a `.. include::`. I've already suggested using `guilabel` in the other one, and noticed that I have to suggest it here as well 😄

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+
+User setup
+~~~~~~~~~~
+
+Using this setup,
+all users who have access to the configured Google Workspace will be granted a subset of permissions on your organization automatically on account creation.
+.. tip::
+
+   You can still add outside collaborators and manage their access. Organizations may for instance wish to give read-access to onboarding documents or grant temporary write access to external consultants.
```
Comment (humitos) [inline docs/user/guides/setup-single-sign-on-google-email.rst:34]: This tip tells you about something but does not tell you where to get more information about this.

There is a `seealso` after this tip, but it's not _inside_ which makes it seem unrelated to this tip.

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 19]
```diff
@@ -0,0 +1,79 @@
...
+------------
+
+By default, users that sign up with a Google account do not have any permissions over any project.
+However, you can define which teams users matching your company's domain email address will auto-join when they sign up.
+
+You can enable this feature in your organization:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > > :guilabel:`Settings` > :guilabel:`Authorization`
+* Select :guilabel:`Google` as the *Provider*
+* Specify your Google Workspace domain as the *Domain*.
```
Comment (humitos) [inline docs/user/guides/setup-single-sign-on-google-email.rst:51]: We should make usage of `guilabel` here as well. BTW, did we already agreed on this? I thought we did, but I may be confused.

**Review by humitos — COMMENTED** [review]
A few and small comments 👍🏼

File: docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/guides/setup-single-sign-on-github-gitlab-bitbucket.rst:32]: thanks @humitos - now I think it reads without the scary ambiguity :)

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 19 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: Looks like Sphinx actually has `:menuselection:`! Perhaps we should also be using this? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection

Comment (humitos) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: Looks good, yeah. I saw that @benjaoming mentioned in the style guide PR as well 👍🏼

File: docs/user/_includes/organization-permissions.rst [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+
+To change your :doc:`Organization's </commercial/organizations>` settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the :guilabel:`Owners` section on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
```
Comment (humitos) [inline docs/user/_includes/organization-permissions.rst:14]:
```suggestion
If you'd like to modify this setting and are not an owner,
```

File: docs/user/commercial/single-sign-on.rst [hunk 21]
```diff
@@ -1,15 +1,17 @@
-Single Sign-On
-==============
+Choosing a Single Sign-On (SSO) approach for your organization
+==============================================================
 
 .. include:: /shared/admonition-rtd-business.rst
 
-Single sign-on is supported on |com_brand| for all users.
+Single sign-on an optional feature on |com_brand| for all users.
```
Comment (humitos) [inline docs/user/commercial/single-sign-on.rst:6]:
```suggestion
Single sign-on is an optional feature on |com_brand| for all users.
```

File: docs/user/commercial/single-sign-on.rst [hunk 22]
```diff
@@ -18,123 +20,40 @@ Users can log out by using the :ref:`Log Out <versions:Logging out>` link in the
...
+------------------------------------------------
 
-Using an identity provider that supports authentication and authorization allows you to manage
-who has access to projects on Read the Docs, directly from the provider itself.
+Using an identity provider that supports authentication and authorization allows organization owners to manage
+who has access to projects on Read the Docs,
+directly from the provider itself.
 If a user needs access to your documentation project on Read the Docs,
-that user just needs to be granted permissions in the VCS repository associated with the project.
+that user just needs to be granted permissions in the git repository associated with the project.
```
Comment (humitos) [inline docs/user/commercial/single-sign-on.rst:30]:
```suggestion
that user just needs to be granted permissions in the Git repository associated with the project.
```

🤷🏼

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 19 above)
Comment (humitos) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: I've never used it, so I'm not sure how it renders or even if our theme supports it 😄

Comment (benjaoming) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: The only thing `:menuselection:` doesn't help us with is the seperator. Sphinx docs suggest `>` and I think that sounds good (but then they use `-->` in the example which seems oddly long).

File: docs/user/commercial/single-sign-on.rst (hunk 12 above)
[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/commercial/single-sign-on.rst:40]: > It seems that is what we're discussing?

Isn't there a not-so-SSO variation where an organization can have users that sign up with "@the-organization.tld" be automatically enrolled in their organization?

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 19 above)
Comment (benjaoming) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: > I've never used it, so I'm not sure how it renders or even if our theme supports it 

If it doesn't support it, we are currently doing new releases, so we should be able to add an issue and fix that quite quickly :)

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+
+Enabling SSO
+------------
+
+By default, users that sign up with a Google account do not have any permissions over any project.
+However, you can define which teams users matching your company's domain email address will auto-join when they sign up.
+
+You can enable this feature in your organization:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > > :guilabel:`Settings` > :guilabel:`Authorization`
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:35]:
```suggestion
* Navigate to :menuselection:`"username dropdown" ⬇️ > Organizations > "Organization Name" > Settings > Authorization`
```

Writing this is actually difficult!

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 24]
```diff
@@ -0,0 +1,65 @@
...
+* Select :guilabel:`Google` as the *Provider*
+* Specify your Google Workspace domain as the *Domain*.
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:37]:
```suggestion
* Specify your Google Workspace domain in the :guilabel:`Domain` input field and press :guilabel:`Save`.
```

File: docs/user/guides/setup-single-sign-on-google-email.rst [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+Enabling SSO
+------------
+
+By default, users that sign up with a Google account do not have any permissions over any project.
+However, you can define which teams users matching your company's domain email address will auto-join when they sign up.
+
+You can enable this feature in your organization:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > > :guilabel:`Settings` > :guilabel:`Authorization`
+* Select :guilabel:`Google` as the *Provider*
```
Comment (benjaoming) [inline docs/user/guides/setup-single-sign-on-google-email.rst:36]:
```suggestion
* Select **Google** in the :guilabel:`Provider` drop-down.
```

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 19 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst:37]: This is addressed in the style guide now 👍

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 23 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst:35]: Looks easier than before? I don't think we need the quotes though?

File: docs/user/commercial/single-sign-on.rst (hunk 12 above)
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/commercial/single-sign-on.rst:40]: No, I don't believe so.

File: docs/user/guides/setup-single-sign-on-google-email.rst (hunk 19 above)
Comment (ericholscher) [reply docs/user/guides/setup-single-sign-on-google-email.rst]: This is how the two render:

Screenshot 2022-12-15 at 3 08 32 PM

Basically `-->` gets a nicer glyph. Both aren't styled though, and should probably be styled as the same as `:guilabel:` 🙃

**Comment by ericholscher** [comment]
Should be ready to merge if folks want to 👍 it.

I actually wanted to get this in front of a customer today, so now I'm gonna be motivated to get `diatasis/main` merged into `main` :)

**Comment by benjaoming** [comment]
Yes, everything resolved :100:

**Comment by ericholscher** [comment]
🎉
